config/routes.rb

```diff
@@ -7,5 +7,7 @@
   patch '/profile', to: 'users#update'
   get '/profile', to: 'users#profile', as: 'user_profile'
   patch '/update_profile_picture', to: 'users#update_profile_picture', as: 'update_profile_picture'
+  get 'used-cars/sell', to: 'used_cars#index'
```

Comment:
This route was already added in Tron-9 ticket here is the pull request for that =>https://github.com/Hassan0112/Trons/pull/10/files#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5, why that change is also reflecting into this PR